extensions/wikia/FacebookClient/FacebookMapModel.class.php

```diff
@@ -127,13 +142,6 @@ public static function getUserMapping( $wikiaUserId, $facebookId ) {
 			throw new FacebookMapModelException( '', self::ERROR_WIKIA_USER_ID_MISMATCH );
 		}
 
```

Comment:
This is overly constrained and is the specific fix for this ticket.  Look at this page for reasons why there can be more than one Wikia ID in the mapping table:

https://one.wikia-inc.com/wiki/UConn/Product/Facebook_Connect/Mapping_Changes#Examples